sim/core/stats/stats.go

```diff
@@ -444,10 +444,14 @@ func NewPseudoStats() PseudoStats {
 type TargetPseudoStats struct {
 	BonusMeleeAttackPower  float64
 	BonusRangedAttackPower float64
-	BonusCritRating        float64 // Bonus crit for all types of hits.
+
+	BonusCritRating float64 // Bonus crit for all types of hits.
 
 	BonusMeleeHitRating float64 // Imp FF
 
+	BonusPhysicalDamageTaken float64 // Gift of Arthas, Hemorrhage
+	BonusHolyDamageTaken     float64 // Judgement of the Crusader
+
 	DamageTakenMultiplier float64 // All damage
 
 	PhysicalDamageTakenMultiplier float64
```

Comment:
The various "XXXDamageTakenMultiplier" (and similar) could be turned into a SpellSchool-indexed slice. This would require to move stats into core (or the SpellSchool somewhere else), however, to not get an import cycle.

Any opinion on this?